src/components/Book/Book.js

```diff
@@ -1,73 +1,42 @@
-import React, { useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { CircularProgressbar } from 'react-circular-progressbar';
-import { getBooks, deleteBook } from '../../redux/books/books';
-import 'react-circular-progressbar/dist/styles.css';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../../redux/books/books';
+import BookForm from '../BookForm/BookForm';
```

Comment:
- [ ] According to the project you should  Fetch your books from Bookstore API on when the component has rendered with` useEffect` . Please fix it 👍